ClientApp/viking-vault/src/ProfilePicture.css

```diff
@@ -2,17 +2,16 @@
 {
     position: relative;
     float: left;
-    margin: 3%;
-    width: 20%;
-    height: 100%;
+    margin: 5%;
+    width: 150px;
+    height: 150px;
+    overflow: hidden;
+    border-radius: 50%;
 }
+
 #ProfilePicture
```

Comment:
Please use the standard CSS ids and class naming which you use in your application, it's `kebab-case` I think?

Reply:
Where were you when the first guy passed the pull request? :)))